locales/te-IN

```diff
@@ -0,0 +1,21 @@
+title = ఫైర్ఫాక్స్  క్వాంటం విడుదల ప్రచారం
+project-name = ఫైర్ఫాక్స్ క్వాంటం విడుదల
+main-title =  ఫైర్ఫాక్స్ తిరుగుబాటు!
+main-description = క్రొత్త  ఫైర్ఫాక్స్ గురించి పదం వ్యాప్తి చేయడానికి మాకు సహాయం చెయ్యండి! మంగళవారం 14, 14:00 UTC , మీకు కొత  ఫైర్ఫాక్స్లొ నచిన వాటి గురించి #newfirefox ఉపయొగించి ట్వెట్ చెయండి లెదా ఫైర్ఫాక్స్  ట్వీట్లను రీట్వీట్ చెయండి 
+thunderclap-cta = ఫైర్ఫాక్స్ క్వాంటంకు మద్దతు తెలపండి!
+thunderclap-link = https://www.thunderclap.it/projects/64595-firefox-quantum-launch-day
+official-tweet-cta = అధికారిక ట్వీట్ని రీట్వీట్ చెయండి 
+tweet-1 = #newfirefox నెడె విడుధలైంధి మరియు ఇది ముందు కన్నా రెండు రెట్లు వేగంగా ఉంది!💥 Download it: https://mzl.la/newfirefox
+tweet-2 = #newfirefox ___________ కన్నా వేగంగా ఉంది.ధన్యవాదాలు @Firefox!
+tweet-3 = కొత @Firefox ఇక్కడ ఉంది మరియు ఇది వేగవంతం.మంచి కొరకు. #newfirefox ను ఇక్కడ పొందండి: https://mzl.la/newfirefox
+own-tweet-cta = స్వంత టెక్స్ట్ను ట్వీట్ చేయండి
+hashtag = #newfirefox
+github = గిఠ్హబ్
+legal = చట్టపరమైనవి
+license = లైసెన్సు
+guidelines = మార్గదర్శకాలు
+mozilla-description =
+మొజిల్లా ప్రపంచవ్యాప్త లాభాపేక్షలేనిది, మీ ఆన్ లైన్ అనుభవాన్ని నియంత్రించి, భవిష్యత్ను రూపొందించడంలో అంకితం చేయబడింది
+  ప్రజా మంచి కోసం వెబ్. మమ్మల్ని సందర్శించండి
+  <a href="https://mozilla.org" target="_blank">mozilla.org</a>
+official-tweet-id = 928033132908986369
```

Comment:
as this is an old tweet and was only used as an example until now, I'm in the process of removing it. Could you please set this string to "tbd"? I will update this tomorrow for all language files once the official tweet is out.

Thanks!

Reply:
Done ! please check.